cfg/sources.js

```diff
@@ -0,0 +1,34 @@
+export const SOURCES = Object.freeze({
```

Comment:
`source` sounds a bit too generic to me - is there somethign slightly narrower we could get happy with? i worry (a) there are other things in our infra that also touch `cbonnoit.github.io` that also have a sense of a source (I'm thinking of a `TranscriptSource` and `AudioSource`) and (b) even for attribution, you might be able to think up multiple distinct attributes around sourcing, while i think with this attribute you really only intend to be enumerating the 'channel' in a marketing sense through which the users arrived.

anyway, how about `traffic_channel`? or do you have any other ideas?

Reply:
called attribution_channel